org.kde.KStyle.Kvantum.json

```diff
@@ -0,0 +1,48 @@
+{
+    "id": "org.kde.KStyle.Kvantum",
+    "branch": "5.12",
+    "runtime": "org.kde.Platform",
+    "build-extension": true,
+    "sdk": "org.kde.Sdk",
+    "runtime-version": "5.12",
+    "separate-locales": false,
+    "appstream-compose": false,
+    "finish-args": [
```

Comment:
These do nothing so just remove them. You have to convince the KDE runtime to add them.